internal/generator/log.go

```diff
@@ -93,6 +97,11 @@ var (
 		"[DEBUG] plugin/errors: 2 kubernetes.default.svc. A: read udp 1.1.1.1:54608->1.1.1.1:53: i/o timeout",
 		"D0426 20:39:28.065697       1 scheduler.go:599] error selecting node for pod: running \"VolumeBinding\" filter plugin for pod \"eric-data-document-database-pg-1\": pod has unbound immediate PersistentVolumeClaims",
 	}
+
+	//go:embed samples_audit.txt
+	auditSamplesRaw string
+
+	auditSamples = strings.Split(auditSamplesRaw, "\n")
```

Comment:
I think you need to trim the last element from the slice, because the input file _correctly_ has an EOL at the end there's a third empty string present.

Reply:
trimmed and added more samples